atomic_reactor/plugin.py

```diff
@@ -45,6 +45,10 @@ class BuildCanceledException(Exception):
     """Build was canceled"""
 
 
+class UnknownPlatformException(Exception):
```

Comment:
This seems out of scope here. Should it be defined in `build_orchestrate_build.py`?